theme/less/_header.less

```diff
@@ -0,0 +1,64 @@
+header
+{
+  box-shadow: 0px 8px 15px rgba(0, 0, 0, 0.06);
+}
+
+header label select
+{
+  color: @secondary;
+  font-size: 24px;
+  border: 0;
+  cursor: pointer;
+  margin: 25px;
+  appearance: none;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  background: url(/images/language-arrow.svg) transparent no-repeat 90% 50%;
```

Comment:
The arrow is seem to be broken on my side, am I missing something?

<img width="179" alt="Screen Shot 2020-08-31 at 8 17 15 PM" src="https://user-images.githubusercontent.com/716969/91747540-0b5ec180-ebc7-11ea-9093-68050e4bb2d0.png">

Reply:
It was from margin of label select option. it works different for chrome and Firefox. On FireFox there is an issue with touching options border to the window border. I tried to solve it by adding margin to label select option but it broke arrow in chrome.

Reply:
That's quite strange that it works in Firefox, because background-image only applies to the content and padding, according to the box model.

Reply:
Meeting notes: We agreed to check if it possible to move arrow into a separate tag(ex: after), if not we will use paddings.

Reply:
In order to use pseudo elements on `select` we need to use a wraper. I did not found any official documentation, but [here](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements) and in other place this solution is mentioned.

Reply:
What about https://cmints.io/ solution?

Reply:
BTW in order to test RTL direction you can add `dir="rtl"` attribute to the `<html>` element.

Reply:
Done. I witched to http://cmints.io/ version of language selector. Added dir="rtl" to html.

Reply:
There was miss-understading:

by cmints.io solution, I meant solution with a absollutely positions label and/or selectbox, but if there is a simpler solution please go for it. Regarding using context menu for the desktop version, let's keep it for a separate PR, in order to keep current PR simple enough for now.

Reply:
I can not recall padding solution. But I chose CmintS.io solution.

Reply:
> I can not recall padding solution

It was your first solution with modified select tag, but using paddings instead of margins. Current solution has a drawback of focused item not being highlighted.